### PR TITLE
add news item

### LINKED ISSUE #1214: news for the cookiecutting activity
### Problem
@sbillinge merged to main without news thinking there may already be a news there but isn't  @tinatn29 please could we have a news for the whole activity.  The wording is somewhere either in teh cookiecutting instructions, or maybe better, look in the changelog of one of the released packages.

### PR DESCRIPTION
**Pull request by tinatn29**
fix #1214. I added `cookie-news.rst` to the `news/` folder per cookiecutting instruction. Let me know if we need anything else here!

### CODE REVIEW

File: news/cookie-news.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* Repo structure modified to the new diffpy standard
```
Comment (bobleesj) [inline news/cookie-news.rst:19]: Maybe need to have "cookiecutter" in the news? since we will be mostly use that term when we revisit?

Comment (tinatn29) [reply news/cookie-news.rst:19]: "Repo structure modified to the new diffpy (cookiecutter) standard"? And we should update the instruction on here too? https://github.com/Billingegroup/cookiecutter/tree/main

Comment (bobleesj) [reply news/cookie-news.rst]: Yes, if you could make an issue, i will address it

Comment (tinatn29) [reply news/cookie-news.rst]: I created an issue in cookiecutter repo: https://github.com/Billingegroup/cookiecutter/issues/185

**Pull request merged**